### DISCUSSION TITLE
LXQt on matrix

### DISCUSSION THREAD

**Discussion by stefonarch** [lxqt/lxqt#1923] [General]
Maybe only few people know that  there exists a  room `#lxqt:matrix.org`   for over 4 years now, created by @paulolieuthier 
Clients:
 * https://element.io/
 * https://github.com/Nheko-Reborn/nheko
 * https://github.com/quotient-im/Quaternion

**Comment by yan12125** [comment]
Is the bridge between Matrix and IRC finished? If so, I'm +1 on declaring the Matrix room as official.

The next steps might be adding the Matrix link to the topic of #lxqt IRC channel and adding the Matrix link besides the IRC one on lxqt-project.org website

Update: https://github.com/lxqt/lxqt.github.io/pull/57 covers part of the steps mentioned above :+1:

**Reply by telmobss1** [reply]
I saw that they were trying this, I hope you can, that way we will have irc, matrix and telegram all connected

**Reply by wxl** [reply]
I can confirm that IRC, Telegram, and Matrix are now linked. It seems we needed to create a new room in Matrix but we reused the old alias, so the address as mentioned above is still valid.

**Comment by stefonarch** [comment]
@paulolieuthier There is some spam recently, can you check?

**Reply by paulolieuthier** [reply]
Just made you a moderator.

**Reply by stefonarch** [reply]
Thanks!
